docs/API_SPECS_DEVICE_MODEL_SCRIPTS.md

```diff
@@ -0,0 +1,183 @@
+API specifications - DeviceModelScripts
+================================
```

Comment:
do you think we should have some document/text explaining how these models are used? e.g. why would one create a device model?

Reply:
Done.